gradio/_simple_templates/simpleimage.py

```diff
@@ -0,0 +1,106 @@
+"""gr.Image() component."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any
+
+from gradio_client.documentation import document, set_documentation_group
+
+from gradio.components.base import Component
+from gradio.data_classes import FileData
+from gradio.events import Events
+
+set_documentation_group("component")
+
+
+@document()
+class SimpleImage(Component):
+    """
+    Creates an image component that can be used to upload images (as an input) or display images (as an output).
+    Preprocessing: passes the uploaded image as a {str} filepath.
+    Postprocessing: expects a {str} or {pathlib.Path} filepath to an image and displays the image.
+    Examples-format: a {str} local filepath or URL to an image.
+    """
+
+    EVENTS = [
+        Events.clear,
+        Events.change,
+        Events.upload,
+    ]
+
+    data_model = FileData
+
+    def __init__(
+        self,
+        value: str | None = None,
+        *,
+        label: str | None = None,
+        every: float | None = None,
+        show_label: bool | None = None,
+        show_download_button: bool = True,
+        container: bool = True,
+        scale: int | None = None,
+        min_width: int = 160,
+        interactive: bool | None = None,
+        visible: bool = True,
+        elem_id: str | None = None,
+        elem_classes: list[str] | str | None = None,
+        render: bool = True,
+    ):
+        """
+        Parameters:
+            value: A PIL SimpleImage, numpy array, path or URL for the default value that SimpleImage component is going to take. If callable, the function will be called whenever the app loads to set the initial value of the component.
+            label: The label for this component. Appears above the component and is also used as the header if there are a table of examples for this component. If None and used in a `gr.Interface`, the label will be the name of the parameter this component is assigned to.
+            every: If `value` is a callable, run the function 'every' number of seconds while the client connection is open. Has no effect otherwise. Queue must be enabled. The event can be accessed (e.g. to cancel it) via this component's .load_event attribute.
```

Comment:
nit: We're removing `Queue must be enabled` in another PR

Reply:
will remove in the other PR after this is merged in.